queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+ SELECT * FROM animals WHERE name LIKE '%mon%';
+ SELECT name FROM animals WHERE date_of_birth>= '2016-01-01' AND date_of_birth<= '2019-01-01';
+ SELECT name FROM animals WHERE neutered IS TRUE AND escape_attempts<3;
+ SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu';
+ SELECT name,escape_attempts FROM animals WHERE weight_kg<10.5;
```

Comment:
- [ ] Kindly make sure to select any animal that weighs more than `10.5kg` not less.